examples/objective-c/helloworld/main.m

```diff
@@ -42,6 +43,7 @@
 int main(int argc, char * argv[]) {
   @autoreleasepool {
     [GRPCCall useInsecureConnectionsForHost:kHostAddress];
+    [GRPCCall usePrimaryUserAgent:@"HelloWorld/1.0" forHost:kHostAddress];
```

Comment:
Do we want different user agents for different hosts? I would think the use case of setting it once for all RPCs is more common, so we could offer `+[GRPCCall usePrimaryUserAgent:]` and later on, if someone asks for it, the overload for a specific host.